odoo/tools/translate.py

```diff
@@ -387,6 +390,9 @@ def is_text(term):
 
 xml_translate.term_adapter = xml_term_adapter
 
+TRANSLATE_FUNCS['html_translate'] = html_translate
+TRANSLATE_FUNCS['xml_translate'] = xml_translate
```

Comment:
if some customized modules want thier own customized translate methods.
They should
1. import the `TRANSLATE_FUNCS` and register the method
2. inherit `ir.model.fields` and add selection options to the `translate` field

If we don't want to support the customzied translate function, we could use a readonly dict